src/api/utils.js

```diff
@@ -80,7 +80,7 @@ const checkOptions = (options) => {
  *
  * @return trimmed object
  */
-function trimObj(obj) {
+export function trimObj(obj) {
```

Comment:
Let's make these two functions `const` arrow functions while we're making exports consistent.